<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSearch</class>
 <widget class="QDialog" name="AdvancedSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Search</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/**
 * Blue: rgb(61, 149, 245);
 * Hover Blue: rgb(13, 122, 242);
 * Light Hover Blue: rgb(187, 225, 250);
 * Red: rgb(242, 22, 43);
 * Hover Red: rgb(203, 11, 29);
 * Green: rgb(82, 224, 102);
 * Yellow: rgb(253, 214, 52);
 * Hover Yellow:  rgb(253, 205, 36);
 * Hover Green: rgb(39, 216, 63);
 * Light Gray: rgb(200, 200, 200);
 * Gray : rgb(239, 239, 239);
 * White: rgb(248, 248, 248);
 */

#AdvancedSearch {
	background-color: rgb(239, 239, 239);
	border-radius: 10px;
}

#searchButton {
	border-radius: 15px;
}

#closeButton {
	background-color:  rgb(242, 22, 43);
	border-radius: 15px;
}

#closeButton:hover {
	background-color: rgb(203, 11, 29);
}

QRadioButton::indicator {
    border: 2px solid rgb(61, 149, 245);
	width: 24px;
	height: 24px;
	border-radius: 14px;
    background: rgb(255, 255, 255);
}
QRadioButton::indicator:hover {
    border: 2px solid rgb(13, 122, 242);
}
QRadioButton::indicator:checked {
	border: 2px solid rgb(13, 122, 242);
    background: rgb(61, 149, 245);
	background-image: url(:/icons/checked.png);
}

.QLabel {
	background-color: rgb(239, 239, 239);
}

QLineEdit {
	background-color: rgb(248, 248, 248);
	border-radius: 5px;
	border: 2px solid rgb(200, 200, 200);
}

QLineEdit:hover {
	border: 2px solid rgb(187, 225, 250);
}

QLineEdit:focus {
	border: 2px solid rgb(13, 122, 242);
}

QPushButton {
	background-color: rgb(61, 149, 245);
	color: rgb(248, 248, 248);
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(13, 122, 242);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="senderLabel">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sender:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="senderLineEdit">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>someone@example.com</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="receiverLabel">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Receiver:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="receiverLineEdit">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>someone@example.com</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Search Type:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="normalSearchRadioButton">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Normal</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QRadioButton" name="andSearchRadioButton">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>And</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QRadioButton" name="orSearchRadioButton">
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Or</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="closeButton">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/cross.png</normaloff>:/icons/cross.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="searchButton">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
